src/pages/en/guides/cms/storyblok.mdx

```diff
@@ -368,29 +366,54 @@ export default defineConfig({
 
 ### Generating pages
 
+To create a route for a specific `page`, you can fetch its directly from the Storyblok API and pass it to the `Page` component. Create an `index.astro` file in `src/pages` to render the `home` page:
+
+```astro title="src/pages/index.astro" {7,8,15} 
+---
+import { useStoryblokApi } from '@storyblok/astro'
+import BaseLayout from '../layouts/BaseLayout.astro'
+import Page from '../storyblok/Page.astro';
+
+const storyblokApi = useStoryblokApi();
+const { data } = await storyblokApi.get('cdn/stories/home');
+const content = data.story.content;
+---
+<html lang="en">
+  <head>
+    <title>Storyblok & Astro</title>
+  </head>
+  <body>
+    <Page blok={content} />
+  </body>
+</html>
+```
+
+To dynamically render all of the blog posts, create a dynamic route. This approach varies depending on whether you're using **static site generation** (the default) or **server-side rendering**.
+
 #### Static site generation
 
 If you are using Astro's default static site generation, you will use [dynamic routes](/en/core-concepts/routing/#dynamic-routes) and the `getStaticPaths` function to generate your project pages.
 
-Create a new file in the `src/pages` directory called `[...slug].astro` and add the following code:
+Create a new file in a `src/pages/blog` directory called `[...slug].astro` and add the following code:
```

Comment:
```suggestion
Create a new directory `src/pages/blog/` and add a new file called `[...slug].astro` with the following code:
```
If the directory doesn't exist, probably good practice to instruct the reader to create the directory.